spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -2,6 +2,14 @@
 
 RSpec.describe Broadcasts::WelcomeNotification::Generator, type: :service do
   describe "::call" do
+    let(:receiving_user) { create(:user) }
+    let(:user) { create(:user) }
```

Comment:
What is the difference between `receiving_user` and `user` here?

Reply:
Ok, looking at the tests a bit closer, I think I see what you're trying to do here. It might make more sense to just create a `user` object here, and then, inside of the context block for sending the welcome thread (line 20), you use a `before` block and do the work of creating the welcome thread and assigning the comment to the user.

Right now, as it stands, it's hard to tell which of these two user objects we should be testing against. Renaming these variables so that it becomes clear which user is associated with the welcome thread + welcome comment explicitly should help out here! (I think using a `before` block is an idea worth exploring here)

Reply:
Okay, I think I understand what you're suggesting and have changed this!